docs/Custom-brimcap-load-Config.md

```diff
@@ -0,0 +1,434 @@
+export LD_LIBRARY_PATH="/usr/local/lib"
+```
+
+## Example Configuration
+
+As we did with Zeek and Suricata, we create a wrapper script to act as our
+Brimcap analyzer. It works in two phases, first creating binary NetFlow records
+and then converting them to CSV. `nfpcapd` only accepts a true pcap file input
+(not a device like `/dev/stdin`), so we first store the incoming pcap in a
+temporary file. We also use GNU `head` to trim out some trailing summary lines
+that `nfdump` appends by default which would trip up the Zed CSV reader.
+
+```
+$ cat nfdump-wrapper.sh 
+TMPFILE=$(mktemp)
+cat - > "$TMPFILE"
+nfpcapd -r "$TMPFILE" -l .
+rm "$TMPFILE"
+for file in nfcapd.*
+do
+  nfdump -r $file -o csv | head -n -3 | zq -i csv -f ndjson - > ${file}.ndjson
+done
+```
+
+This script is called from our Brimcap config YAML, which includes a
+`globs:` setting to target only the CSV files and also a Zed shaper to apply
+rich data types.
+
+```
+$ cat nfdump.yml 
+analyzers:
+  - cmd: nfdump-wrapper.sh
+    # The globs being set to "*.ndjson" is a workaround for the fact that our
+    # current JSON reader won't accept much data and we can't tell
+    # brimcap load to expect CSV input, so we're postprocessing in the
+    # nfdump-wrapper.sh script with zq to turn the CSV back into NDJSON.
+    globs: ["*.ndjson"]
+    shaper: |
+      type netflow = {
+        ts: time,
+        te: time,
+        td: duration,
+        sa: ip,
+        da: ip,
+        sp: uint16,
+        dp: uint16,
+        pr: string,
+        flg: string,
+        fwd: bytes,
+        stos: bytes,
+        ipkt: uint64,
+        ibyt: uint64,
+        opkt: uint64,
+        obyt: uint64,
+        \in: uint64,
+        out: uint64,
+        sas: uint64,
+        das: uint64,
+        smk: uint8,
+        dmk: uint8,
+        dtos: bytes,
+        dir: uint8,
+        nh: ip,
+        nhb: ip,
+        svln: uint16,
+        dvln: uint16,
+        ismc: string,
+        odmc: string,
+        idmc: string,
+        osmc: string,
+        mpls1: string,
+        mpls2: string,
+        mpls3: string,
+        mpls4: string,
+        mpls5: string,
+        mpls6: string,
+        mpls7: string,
+        mpls8: string,
+        mpls9: string,
+        mpls10: string,
+        cl: float64,
+        sl: float64,
+        al: float64,
+        ra: ip,
+        eng: string,
+        exid: bytes,
+        tr: time
+      }
+      // The leading "put tr=time(tr)" is a workaround to zed/2670
+      put tr=time(tr) | put . = shape(netflow)
+```
+
+Putting it all together, we can test it by creating a new pool and then running
+`brimcap load` to import a sample pcap.
+
+```
+$ /opt/Brim/resources/app.asar.unpacked/zdeps/zed api new testpool2
+$ /opt/Brim/resources/app.asar.unpacked/zdeps/brimcap load -root "$HOME/.config/Brim/data/brimcap-root" -config nfdump.yml -p testpool2 sample.pcap
+```
+
+Our pool is now ready to be queried in Brim.
+
+![NetFlow Pool](media/NetFlow-Pool.png)
```

Comment:
Note the histogram is empty, which is unsurprising since I think we're still wired to only populate this if the field named `_path` is present. We've discussed in the past that it might be helpful to be able to specify a primary "segmentation axis" for the data in a pool (e.g. `_path` for Zeek) that could be used to make stacked bars when present, and in its absence, just give non-stacked "count" bars?